vm-virtio/src/net.rs

```diff
@@ -658,15 +508,17 @@ impl VirtioDevice for Net {
             }
             self.queue_evts = Some(tmp_queue_evts);
 
-            let rx_queue = queues.remove(0);
-            let tx_queue = queues.remove(0);
-            let rx_queue_evt = queue_evts.remove(0);
-            let tx_queue_evt = queue_evts.remove(0);
+            let mut queues_v = Vec::new();
```

Comment:
Do the names `queues` and `queue_evts` clash? We don't normally use any suffixes like this.

Reply:
The variable naming isis inherited from the activate method which has queues and queue_evts, Do you have any better way to re-name them?

Reply:
Do you need to create new variables? Just modify the handler_run to use `queues.split_off(0)` etc?

Reply:
Hmm, it's not allowed to directly pass queues.split_off to handler.run() for value move happens. It's still required to clone it to a new vector.

Reply:
split_off does create a new vector.

Reply:
Refer to page https://doc.rust-lang.org/std/vec/struct.Vec.html#method.split_off, the original self contains data [0, at), the returned Self has [at, len). If it only needs to split_off queues once, this should be a more convenient way to get the sub vector, however, it's a loop to get all queues that two queues every time, based on the split_off explanation, it seems not to be a proper method for our case.